language/attributes.xml

```diff
@@ -297,7 +297,7 @@ dumpMyAttributeData(new ReflectionClass(Thing::class));
     Es ist zwar nicht unbedingt erforderlich, für jedes Attribut eine eigene
     Klasse zu erstellen, aber es ist empfehlenswert. Im einfachsten Fall wird
     nur eine leere Klasse benötigt, in der das Attribut
-    <literal>#[Attribut]</literal> deklariert wird, das mit einer use-Anweisung
+    <literal>#[Attribute]</literal> deklariert wird, das mit einer use-Anweisung
```

Comment:
Hier war das `#[Attribut]` absichtlich ohne "e" um auszudrücken, dass es ein beliebiger Name sein kann (korrigiere mich bitte, falls ich falsch liege).

Reply:
I understand it to be the declaration and not the attribute itself (which is called `MyAttribute` and not "Attribut" in the code example). So you must use the declaration of `#[Attribute]` to flag `MyAttribute` as attribute.

Reply:
OK, thanks. Even if it wasn't the case, it won't hurt either. ;)